src/app/api/branching/branch/delete/route.js

```diff
@@ -0,0 +1,65 @@
+import { verifySignatureAppRouter } from '@upstash/qstash/dist/nextjs';
+import { NextResponse } from 'next/server';
+import * as yup from 'yup';
+
+const requestSchema = yup.object({
+  branchId: yup.string().required('Branch ID is required'),
+  retries: yup.number().integer().min(0).max(3).default(0),
```

Comment:
since we're using Upstash QStash, we'd not need `retries` to be handled as part of the requests schema.